### PR TITLE
Integrated native lib v1.3.0

### PR DESCRIPTION
**Pull request by Vovchyk**
## Description
This upgrades native library to version 1.3.0. With that the support of linux/arm64 is being added.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Requires Activation Code (Hard Fork)

*fed:native-lib-130-upgrade*

### CODE REVIEW

**Comment by Vovchyk** [comment]
pipeline:run

**Comment by Vovchyk** [comment]
pipeline:run

**Comment by Vovchyk** [comment]
pipeline:run

File: rskj-core/src/main/java/co/rsk/pcc/altBN128/impls/AbstractAltBN128.java [hunk 1]
```diff
@@ -16,20 +18,36 @@ public abstract class AbstractAltBN128 {
...
-        return init(Utils::isLinux, GoAltBN128::getLoadError);
+    /**
+     * Besides logging information about loaded library, this method has a side effect of enforcing native library loading,
+     * if it's not loaded yet. So that it allows to fail early, if, for instance, the loaded library is not compatible
+     * with the current environment.
+     */
+    public static void init() {
+        AbstractAltBN128 altBn128 = Objects.requireNonNull(create());
+
+        if (Utils.isLinux() && !(altBn128 instanceof GoAltBN128)) {
```
Comment (illuque-iov) [inline rskj-core/src/main/java/co/rsk/pcc/altBN128/impls/AbstractAltBN128.java:29]: Just a minor and picky thing, why have you "moved" logs here? I thing they belong to the place where the "selection" is made, this way you won't need this check and this `instanceof`.

But maybe I'm not getting it right or I'm missing something.

Comment (Vovchyk) [reply rskj-core/src/main/java/co/rsk/pcc/altBN128/impls/AbstractAltBN128.java:29]: the selection depends on the current environment, eg. linux vs mac. And this is not gonna change during execution. So the log message is printed only one time on node start. I'd say pretty much the same as it's done in `Secp256k1.initialize()`

Comment (illuque-iov) [reply rskj-core/src/main/java/co/rsk/pcc/altBN128/impls/AbstractAltBN128.java:29]: Got it now, thanks!

**Review by rmoreliovlabs — APPROVED** [review]

**Pull request merged**